common/cards/boss/hermits/evilxisuma_boss.ts

```diff
@@ -328,7 +328,7 @@ const EvilXisumaBoss: Hermit = {
 				.forEach((card) => destroyCard(game, card))
 			removeImmuneEffects(game, component.slot)
 		})
-		let lastAttackDisabledByAmnesia = false
+		let lastAttackDisabledByAmnesia = NaN
```

Comment:
I don't think using `NaN` is a very good idea. I find its really confusing so it lead to bugs.